Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1520,6 +1520,24 @@ final class LayerTests: XCTestCase {
     }
   }
 
+  func testBiBasicRNN() {
+    let x = Tensor<Float>(rangeFrom: 0.0, to: 0.4, stride: 0.1).rankLifted()
+    let inputs: [Tensor<Float>] = Array(repeating: x, count: 4)
+    let birnn = BiBasicRNN<Float>(BasicRNNCell(inputSize: 4, hiddenSize: 4, seed: (0xFeed, 0xBeef)), mergeMode: .sum)
+    withTensorLeakChecking {
+      let outputs = birnn(inputs)
+      assertEqual(
+        outputs.map { $0.value.squeezingShape(at: 0) }[0],
+        Tensor<Float>(shape: TensorShape([4, 4]), scalars: [
+          0.20775771, 0.20080023, -0.13768704, -0.18534681,
+          0.22666009, 0.30019346, -0.19720285, -0.14683801,
+          0.23758979, 0.32101023, -0.20359215, -0.1787096,
+          0.24337786, 0.3389194, -0.21143384, -0.1675081,
+        ].map { $0 + $0 }),
+        accuracy: 1e-6)
+    }
+  }
+
```

Comment:
I wonder if you've tested bidirectional recurrent layer correctness using a Python reference implementation?

We have an [existing `lstm.py` script](https://github.com/tensorflow/swift-apis/blob/master/Utilities/ReferenceImplementations/lstm.py) for testing LSTM correctness versus Keras. Could you please copy it to `bidirectional_rnn.py` and adapt it to implement the same `Bidirectional` program as the one here in Swift?

Reply:
I did not, but this code is from `testRNN`, only difference is `.map { $0 + $0 }` because merge mode is `sum`. 
I will check the script and adapt it 👍

Reply:
I am making progress on tests, all seems to be ok except that the last row of `gradInputs` does not correspond to the last row of `expectedGradX`.  I am trying to figure out why now.

Reply:
@dan-zheng I finally get the tests working 🎉 But I needed to do some changes (I got inspiration from Keras implementation), could you please check last commit and give me feedback? Thank you.